Implemented welcome pager

Meisolsson opened the pull request:
#872 Was asking for a welcome pager. This design is from the Google Drive app.

![Example](https://cloud.githubusercontent.com/assets/3757908/12012467/501c3c3c-acf5-11e5-968d-d0480b3cc25c.gif)

Thoughts?

larsgrefer commented:
lgtm :+1:

StefMa commented:
Is the first color the same as the "accent"? When yes, it looks good :)
The last color is in my opinion to "dangerous" red. Maybe a little softer red?

Meisolsson commented:
Just took a blue from the Material Design Spec and used paletton's triad mode. And yeah the last one is a bit much, I'll see what I can do about it

Meisolsson commented:
Are the colors better?
![welcome_screen_new](https://cloud.githubusercontent.com/assets/3757908/12018428/9a3faef8-ad62-11e5-8c34-066ee69b6a21.gif)

fadils commented:
awesome!

fadils commented on app/src/main/java/com/github/pockethub/ui/WelcomeActivity.java:
```diff
@@ -0,0 +1,204 @@
+    protected PagerAdapter createAdapter() {
+        return new WelcomePagerAdapter(R.layout.welcome_pager_item,
+                new int[]{R.drawable.waving_android, R.drawable.welcome_app_icon, R.drawable.thumb_up},
+                new String[]{"So a while a go the GitHub app was discontinued and some of us wanted something new...",
```
Can we move these strings into resource instead?

larsgrefer replied:
:+1: Once this is done, i'll provide the german translation

Meisolsson replied:
Yeah, I'll fix that

hzsweers commented:
I'm not sold on the colors yet. Not sure what would be better, but they just feel kind of random right now. What about this?

1. The offwhite github color as the background for the android
2. Our primary grey as the background for the pockethub logo
3. Nice muted green to go with the thumbs up. The red makes it seem like that page is a roadblock to me.

Also fixed travis on master if you want to rebase

Meisolsson commented:
Yeah the last two are kinda weird, but the first one is our accent and it looks good in my opinion. So what about:

1. Our current accent (#3797C7)
2. Our primary color (#3D3D3D)
3. Green (Something like #6AA811)

Meisolsson commented:
Rebased and added new colors.
![welcome_screen_new_colors](https://cloud.githubusercontent.com/assets/3757908/12052405/00b58510-af0f-11e5-8e35-42c58931e34b.gif)

hzsweers commented:
Still not sold on the green in the last pager or the blue and green on the first, so one last idea I'm going to try today:

Offwhite with green android for first page

Accent blue background with white thumbs up.

The color scheme of the app is very grey/blue/white. The green is just too loud for me right now, and the green on blue in the first page just looks a little much. The icons and text should be the accents, not part of the backdrop.

Will report back

hzsweers commented:
How's this?

![screenshot 2016-01-10 00 16 50](https://cloud.githubusercontent.com/assets/1361086/12219949/7d7750f6-b72f-11e5-957a-8d9a67d25d74.png)

StefMa commented:
This is the best looking colors.
I like it.
Am 10.01.2016 5:18 vorm. schrieb "Zac Sweers" <notifications@github.com>:

> How's this?
>
> [image: screenshot 2016-01-10 00 16 50]
> <https://cloud.githubusercontent.com/assets/1361086/12219949/7d7750f6-b72f-11e5-957a-8d9a67d25d74.png>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pockethub/PocketHub/pull/971#issuecomment-170312996>.
>

Meisolsson commented:
Look good, can you post the color values?

hzsweers commented:
I'll send you a patch file on slack that you can apply to your branch
directly
On Sun, Jan 10, 2016 at 2:23 AM Henrik Olsson <notifications@github.com>
wrote:

> Look good, can you post the color values?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pockethub/PocketHub/pull/971#issuecomment-170332351>.
>

Meisolsson commented:
Nice!

Meisolsson commented:
Patch has been applied!
